extractor/src/test/resources/generated-config-all-in-one.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
```

Comment:
Please in name of file "generated" to "expected"
And remove old expected.

Reply:
chane names manually right?

Reply:
I reverted back the maintest.java where i was generating files. do you want me to revert maintest.java now?

Reply:
Done.